packages/discovery-react-components/src/components/CIDocument/utils/details.ts

```diff
@@ -52,7 +52,7 @@ function getContractAttributes(attributes: any): Item[] {
   });
 }
 
-function getAttributesWithinRelation(attributes: any): Item[] {
+export function getAttributesWithinRelation(attributes: any): Item[] {
```

Comment:
why is it exported? Just so it can be tested individually in the tests?

Reply:
It is mainly for testing purposes.  I use this function to get the specific output needed to do the test compare. The alternative would be to mock the return value for that function.